src/Synergism.ts

```diff
@@ -870,6 +870,9 @@ const loadSynergy = async () => {
         Object.defineProperty(window, 'Decimal', {
             value: Decimal
         });
+        if (data && testing) {
+            data.exporttest = false;
+        }
```

Comment:
Last time I pushed this with the intent of making it easier to test builds publicly. Looking at quark/GQ/Oct spending in particular and did not want saves to be ruined or bugs to award free quarks. Not that important in a single player game, but still did not want to expose easy warping and day resets to testers able to migrate back.

This version checks that we're actually testing, and not just non-production.